community-membership.md

```diff
@@ -142,14 +141,14 @@ code quality and correctness, approval is focused on holistic acceptance of a co
 backwards / forwards compatibility, adhering to API and flag conventions, subtle performance and correctness issues,
 interactions with other parts of the system, etc.
 
-**Defined by:** *approver* entry in an OWNERS file in the kubernetes codebase
+**Defined by:** *approvers* entry in an OWNERS file in the kubernetes codebase
```

Comment:
These were written in terms of the kubernetes repo, but now that we're running the automation everywhere, we could generalize this for all repos. I recently counted more than 4000 OWNERS files across our repos.